docs/conceptual/programmers-guide.rst

```diff
@@ -34,10 +34,10 @@ The outer API layer serves as a functional interface for the user to define tens
 between the API objects and the desired functionality. The solution layer encompasses the translation of the input problem parameters into solution candidates, candidate selection, resource management and logging.
```

Comment:
Change title to:
hipTensor programmer's guide

On line 3, add the keyword "cuTensor"

Reply:
> Change title to: hipTensor programmer's guide
> 
> On line 3, add the keyword "cuTensor"

Hi @lpaoletti, just wondering if it is common practice to include cuda library name in rocm document?

Reply:
@CongMa13 - yes, we sometimes note the alternative/parallel library 
My suggestion is to add it to the keywords (line 3). This is used to build the search indexing. 

By the way, the introduction also includes a reference to CUDA:
![image](https://github.com/user-attachments/assets/d1be08df-e995-4826-85a0-026f806ac222)

Reply:
Perfect. Thanks for the explanation. Just want to make sure that everything is complied with the documentation policies.